Command/QueryCommand.php

```diff
@@ -53,9 +53,9 @@ public function execute(InputInterface $input, OutputInterface $output)
         $locale = $input->getOption('locale');
 
         $searchManager = $this->getContainer()->get('massive_search.search_manager');
-        $res = $searchManager->createSearch($query)->index($index)->locale($locale)->execute();
+        $res = $searchManager->createSearch($query)->indexes($index)->locale($locale)->execute();
```

Comment:
You can specify either a single `index` or multiple `indexes`. The query command now supports multiple indexes.